api/TrackDefault.json

```diff
@@ -5,10 +5,10 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/TrackDefault",
         "support": {
           "chrome": {
-            "version_added": null
+            "version_added": true
```

Comment:
This and TrackDefaultList is wrong, it's `RuntimeEnabled=MediaSourceExperimental` in Chromium.

Reply:
I can't seem to find that runtime flag in chrome://flags, should we set it to `false`?